src/util/containers/ContainerManager.ts

```diff
@@ -0,0 +1,27 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+
+/**
+ * Handles everything related to container identifiers and how they relate to other identifiers.
```

Comment:
```suggestion
 * Captures the behavior of container identifiers in a certain storage configuration.
```

to emphasize why there would be multiple.